src/App.jsx

```diff
@@ -21,10 +29,33 @@ function App() {
         wordDisplay.push("_");
       }
     }
+    checkWord(wordDisplay);
     return wordDisplay.toString();
   };
 
   // create additional function to power the
+  
+  // check for amount of guesses
+  const checkGuesses = () => {
+    if(guesses <= 0 || guesses > 10)
+    {
+      setIsAlive(false);
+    };
+  };
+
+  // check for winning condition
+  const checkWord = (letters) => {
+    let word = "";
+    for (let letter of letters)
+    {
+      word += letter;
+    }
```

Comment:
```suggestion
   const word = letters.join("")
```
I think this should also work